src/erldns_zone_cache.erl

```diff
@@ -23,6 +23,7 @@
 
 -include_lib("dns_erlang/include/dns.hrl").
 -include("erldns.hrl").
+-include_lib("stdlib/include/ms_transform.hrl").
```

Comment:
Still used?

Reply:
good catch - the ets:fun2ms/1's calls have been removed.